### PR TITLE
Add translations for modals

### PR DESCRIPTION
**Pull request by lizonp**

### CODE REVIEW

File: app/components/modals/AccessLightWalletPrompt.tsx [hunk 1]
```diff
@@ -30,10 +32,10 @@ const AccessLightWalletForm = (formikBag: FormikProps<IFormValues> & IProps) =>
     <div className="mt-3">
       <Button type="submit" disabled={!formikBag.values.password}>
         {" "}
-        Accept
```
Comment (krzkaczor) [inline app/components/modals/AccessLightWalletPrompt.tsx]: i wouldnt put every button translation into from.button. Please keep it in separate namespace as it is specific to given view.

Comment (Moejoe90) [reply app/components/modals/AccessLightWalletPrompt.tsx]: I thought we agreed to reuse these and call them general components i did the same thing in my PR

File: intl/locales/en-en.json [hunk 2]
```diff
@@ -128,6 +128,11 @@
   "kyc.start.go-to-company": "I represent a company",
   "kyc.start.go-to-personal": "I represent myself",
   "kyc.start.title": "Start your KYC",
+  "modal.button.cancel": "Cancel",
+  "modal.button.dismiss": "Ok",
```
Comment (krzkaczor) [inline intl/locales/en-en.json]: use form.button.ok

File: intl/locales/en-en.json [hunk 3]
```diff
@@ -128,6 +128,11 @@
   "kyc.start.go-to-company": "I represent a company",
   "kyc.start.go-to-personal": "I represent myself",
   "kyc.start.title": "Start your KYC",
+  "modal.button.cancel": "Cancel",
```
Comment (krzkaczor) [inline intl/locales/en-en.json]: use form.button.cancel

**Pull request merged**